### PR TITLE
Admin/exercise confirmation

### PR DESCRIPTION
**Pull request by Ayaka0001**
still in progress

done with:
・frontend
・backend (partly)

still need:
・sort function

### CODE REVIEW

File: resources/views/layouts/admin.blade.php [hunk 1]
```diff
@@ -17,6 +17,7 @@
     @vite(['resources/sass/app.scss', 'resources/js/app.js'])
 
     <link rel="stylesheet" href="{{ asset('css/nav-menu.css') }}">
+    <link rel="stylesheet" href="{{ asset('css/confirmation.css') }}">
 
     <script src="https://kit.fontawesome.com/dbc5b98639.js" crossorigin="anonymous"></script>
```
Comment (keanbillyfernandez) [inline resources/views/layouts/admin.blade.php:22]: Download this kit because it may cause some blocking in other browsers

Comment (Ayaka0001) [reply resources/views/layouts/admin.blade.php:22]: I'm sorry, I'm not sure what I am suppose to download

Comment (keanbillyfernandez) [reply resources/views/layouts/admin.blade.php:22]: @Ayaka0001  - Go to this link https://kit.fontawesome.com/dbc5b98639.js copy and paste all of the code in the screen. Save it in assets folder as .js file

Comment (Ayaka0001) [reply resources/views/layouts/admin.blade.php:22]: is this right?

 inside layouts/admin.blade.php 
  <link rel="stylesheet" href="{{ asset('css/nav-menu.css') }}">
    <link rel="stylesheet" href="{{ asset('css/confirmation.css') }}">
    <link rel="stylesheet" href="{{ asset('js/.js') }}">
</head>

Comment (Ayaka0001) [reply resources/views/layouts/admin.blade.php:22]: <link rel="stylesheet" href="{{ asset('css/nav-menu.css') }}">
    <link rel="stylesheet" href="{{ asset('css/confirmation.css') }}">
    <link rel="stylesheet" href="{{ asset('js/.js') }}">
</head>

Comment (Ayaka0001) [reply resources/views/layouts/admin.blade.php:22]: Screenshot 2024-10-31 at 21 22 26

Comment (Ayaka0001) [reply resources/views/layouts/admin.blade.php:22]: If this is correct, the fontawsome icons are not showing in my pages.

**Pull request merged**